xbmc/pvr/recordings/PVRRecordings.h

```diff
@@ -40,6 +41,8 @@ namespace PVR
     PVR_RECORDINGMAP             m_recordings;
     unsigned int                 m_iLastId;
     bool                         m_bGroupItems;
+    CVideoDatabase               m_db;
+    bool                         m_dbLoaded;
```

Comment:
Doesn't `CVideoDatabase` have an `IsOpen()` method that could be used instead of relying on a boolean?